content/docs/reference-react-component.md

```diff
@@ -126,6 +126,7 @@ When called, it should examine `this.props` and `this.state` and return one of t
 - **Portals**. Let you render children into a different DOM subtree. See the documentation on [portals](/docs/portals.html) for more details.
 - **String and numbers.** These are rendered as text nodes in the DOM.
 - **Booleans or `null`**. Render nothing. (Mostly exists to support `return test && <Child />` pattern, where `test` is boolean.)
+- **Undefined.** Returns undefined when the output is not defined properly.
```

Comment:
`undefined` should be included in the previous line since it renders nothing. "the output is not defined properly" begs the question how output would defined properly. But `undefined` is proper output now.